install/pre-install-payload/scripts/reqs-deploy.sh

```diff
@@ -68,12 +70,12 @@ function install_vfio_gpu_containerd_artefacts() {
 function install_nydus_snapshotter_artefacts() {
 	echo "Copying nydus-snapshotter artifacts onto host"
 
-	install -D -m 755 ${artifacts_dir}/opt/confidential-containers/bin/containerd-nydus-grpc /opt/confidential-containers/bin/containerd-nydus-grpc
-	install -D -m 755 ${artifacts_dir}/opt/confidential-containers/bin/nydus-overlayfs /opt/confidential-containers/bin/nydus-overlayfs
+	install -D -m 755 "${artifacts_dir}"/opt/confidential-containers/bin/containerd-nydus-grpc /opt/confidential-containers/bin/containerd-nydus-grpc
+	install -D -m 755 "${artifacts_dir}"/opt/confidential-containers/bin/nydus-overlayfs /opt/confidential-containers/bin/nydus-overlayfs
```

Comment:
the same here, it's a single argument to `install` so IMO it'd look better to quote it as a whole rather than an individual portion.